.envrc.example

```diff
@@ -6,4 +6,7 @@ export REDIS_INIT="true"
 export GOOGLE_MAX_ATTEMPT="5"
 export BEARER_EXECUTABLE_PATH="./bearer"
 export GITHUB_WORKSPACE="/path/to/bearer/project"
-export SCAN_DIR=/Users/username/OWASP
\ No newline at end of file
+export SCAN_DIR=/Users/username/OWASP
+export BEARER_DISABLE_DEFAULT_RULES=true
+export BEARER_EXTERNAL_RULE_DIR=../bearer-rules/rules
+export BEARER_FORCE=true
```

Comment:
Do we need to do `unset ${!BEARER_*}` when running the tests maybe? (in `run_tests.sh` I would have thought)